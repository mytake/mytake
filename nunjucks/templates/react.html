{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
