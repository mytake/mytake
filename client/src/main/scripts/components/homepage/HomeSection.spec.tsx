import * as React from "react";
import * as renderer from "react-test-renderer";
import HomeSection from "./HomeSection";

test("HomeSection", () => {
  const tree = renderer.create(<HomeSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
