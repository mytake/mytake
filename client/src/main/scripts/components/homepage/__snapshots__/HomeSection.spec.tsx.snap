// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeSection 1`] = `
<section
  className="home__section"
>
  <div
    className="home__section-content"
  />
</section>
`;
